package.json

```diff
@@ -87,6 +87,7 @@
     }
   },
   "prettier": {
+    "arrowParens": "avoid",
```

Comment:
This default changed from v1 to v2, so I added this preference to reduce the number of changes we have to worry about.